word_nerd/flashcards/views.py

```diff
@@ -13,3 +16,28 @@ def get(self, request):
         languages = Language.objects.all().order_by('name')
         serializer = LanguageSerializer(languages, many=True)
         return Response(serializer.data)
+
+
+class AddLanguageAPIView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request):
+        serializer = MyLanguageSerializer(data=request.data)
+
+        if not serializer.is_valid():
+            return Response(
+                {'detail': 'Invalid data.', 'errors': serializer.errors},
+                status=status.HTTP_400_BAD_REQUEST)
```

Comment:
I don't remember what codes we returned previously but `HTTP_422_UNPROCESSABLE_ENTITY` seems most fitting here